src/class-wpml-gutenberg-integration.php

```diff
@@ -152,6 +155,71 @@ private function update_block_translations( $blocks, $string_translations, $lang
 		return $blocks;
 	}
 
+	/**
+	 * @param array $block
+	 *
+	 * @return string
+	 */
+	private function render_block( $block ) {
+		$content = '';
+
+		if ( isset( $block['blockName'] ) ) {
+			$block_type = substr( $block['blockName'], 5 ); // strip core. eg core/paragraph
```

Comment:
@brucepearson, will this method render only block from "core"? In that case, why not `preg_replace( '/^core\//', '', $block['blockName'] )` and remove the comment?

Reply:
@strategio Good idea. I've changed this.
We're only handling `core` now because that seems to be the only type Gutenberg is handling in its parser from what I can see.